form/form.go

```diff
@@ -502,6 +519,15 @@ func BuildField(fileDescriptorSet *descriptor.FileDescriptorSet, msg *descriptor
 		tooltip = ` <a href="#" data-toggle="tooltip" title="` + strings.Replace(help, "\n", " ", -1) + `"><span class="glyphicon glyphicon-question-sign" aria-hidden="true"></span></a>`
 		colon = ""
 	}
+
+	var oneofClass string
+	var oneofClick string
+	if f.OneofIndex != nil {
+		oneofClass += fmt.Sprintf("oneof-%d '+oneofDisabled(json['"+f.GetName()+"'])+'", *f.OneofIndex)
+		action := fmt.Sprintf(`oneofSelect(this, \'oneof-%d\')`, *f.OneofIndex)
+		oneofClick += `onClick="` + action + `" onFocusIn="` + action + `" onChange="` + action + `"`
```

Comment:
I noticed that just onClick wasn't working when selecting the existing value (drag-select). The OnFocusIn and onChange (typing) fix that.